src/cfnlint/data/AdditionalSpecs/Inclusive.json

```diff
@@ -12,22 +12,6 @@
       "IamCertificateId": [
         "SslSupportMethod"
       ]
-    },
-    "AWS::EC2::SecurityGroup.Ingress": {
-      "FromPort": [
```

Comment:
Same example goes for the rest of these.
If you do `-1` the for protocol you can specify just `FromPort`. So this `FromPort` `ToPort` requirement is only a problem when the Protocol is TCP or UDP